cmd/helm/status.go

```diff
@@ -68,6 +68,7 @@ func (s *statusCmd) run() error {
 
 	fmt.Fprintf(s.out, "Last Deployed: %s\n", timeconv.String(res.Info.LastDeployed))
 	fmt.Fprintf(s.out, "Status: %s\n", res.Info.Status.Code)
+	fmt.Fprintf(s.out, "Resources:\n%s\n", res.Info.Status.Resources)
```

Comment:
Thanks! Since kubectl does this, I think there must be a way to have it handle the outputs properly though. If I can't figure it out, I'll use this. But then it would probably make sense to move this to release_server unless we want to have more structured return values.